root/plex-common.sh

```diff
@@ -19,7 +19,9 @@ function getVersionInfo {
     channel=8
   fi
   
-  local url="https://plex.tv/downloads/details/1?build=linux-ubuntu-x86_64&channel=${channel}&distro=ubuntu"
+  local plexBuild="$(cat /plex-build.txt)"
+  local plexDistro="$(cat /plex-distro.txt)"
```

Comment:
Would need similar changes if we have a single file.

Reply:
Agree. 👍